Solutions/CiscoASA/Data Connectors/CiscoASA.JSON

```diff
@@ -0,0 +1 @@
in the selected workspace","innerSteps":[{"title":"1.1 Select or create a Linux machine","description":"Select or create a Linux machine that Microsoft Sentinel will use as the proxy between your security solution and Microsoft Sentinel this machine can be on your on-prem environment, Azure or other clouds."},{"title":"1.2 Install the CEF collector on the Linux machine","description":"Install the Microsoft Monitoring Agent on your Linux machine and configure the machine to listen on the necessary port and forward messages to your Microsoft Sentinel workspace. The CEF collector collects CEF messages on port 514 TCP.\n\n> 1. Make sure that you have Python on your machine using the following command: python --version.\n\n> 2. You must have elevated permissions (sudo) on your machine.","instructions":[{"parameters":{"fillWith":["WorkspaceId","PrimaryKey"],"label":"Run the following command to install and apply the CEF collector:","value":"sudo wget -O cef_installer.py https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_installer.py&&sudo python cef_installer.py {0} {1}"},"type":"CopyableLabel"}]}]},{"title":"2. Forward Cisco ASA logs to Syslog agent","description":"Configure Cisco ASA to forward Syslog messages in CEF format to your Microsoft Sentinel workspace via the Syslog agent.\n\nGo to [Send Syslog messages to an external Syslog server](https://aka.ms/asi-syslog-cisco-forwarding), and follow the instructions to set up the connection. Use these parameters when prompted:\n\n1.  Set \"port\" to 514.\n2.  Set \"syslog_ip\" to the IP address of the Syslog agent.\n\n\n[Learn more >](https://aka.ms/CEFCisco)"},{"title":"3. Validate connection","description":"Follow the instructions to validate your connectivity:\n\nOpen Log Analytics to check if the logs are received using the CommonSecurityLog schema.\n\n>It may take about 20 minutes until the connection streams data to your workspace.\n\nIf the logs are not received, run the following connectivity validation script:\n\n> 1. Make sure that you have Python on your machine using the following command: python --version\n\n>2. You must have elevated permissions (sudo) on your machine","instructions":[{"parameters":{"fillWith":["WorkspaceId"],"label":"Run the following command to validate your connectivity:","value":"sudo wget -O cef_troubleshoot.py https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_troubleshoot.py&&sudo python cef_troubleshoot.py  {0}"},"type":"CopyableLabel"}]},{"title":"4. Secure your machine ","description":"Make sure to configure the machine's security according to your organization's security policy\n\n\n[Learn more >](https://aka.ms/SecureCEF)"}],"metadata":{"id":"31e95c16-9972-4afe-9697-05b9f40f2a58","version":"1.0.0","kind":"dataConnector","source":{"kind":"community"},"author":{"name":"Cisco Systems"},"support":{"name":"Cisco Systems","link":"https://www.cisco.com/c/en/us/support/index.html","tier":"developer"}}}
```

Comment:
Please fix the formatting of this file content. Complete file Json is showing in a single line.

Reply:
done